feature-group-definitions/README.md

```diff
@@ -0,0 +1,63 @@
+# Feature groups
+
+This directory houses the experiment to provide structured definitions groups of web platform features.
+At inception, it consists of this README file only, but it should soon contain data and tools.
+See [ddbeck/common-web-feature-mockup](https://github.com/ddbeck/common-web-feature-mockup/) for more information.
+
+
+## What is a web platform feature?
+
+For the purposes of this project, consider this defintion of a web platform feature:
```

Comment:
On discussion with @atopal, I ought to come back to this to emphasize:

- The descriptive nature of the definition (i.e., that we're trying to capture things which are commonly true of features of the web, not provide a checklist or set of requirements)
- The narrow application of the definition (i.e., we're using this definition as part of an effort to understand web developers, their needs, and the ways they understand the web, which, by necessity, is an approximation of a great many individual perspectives)
- The incompleteness of this definition (i.e., that we're most concerned with the web developer audience and that other definitions might be more useful to specification authors or implementers or technical writers)